pages/members/index.tsx

```diff
@@ -37,51 +57,41 @@ export default class MembersPage extends PureComponent {
     };
 
     list.forEach((member: Member) => {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of